jvm/ml4ir-inference/pom.xml

```diff
@@ -10,6 +10,73 @@
   <artifactId>ml4ir-inference</artifactId>
   <packaging>jar</packaging>
 
+  <profiles>
+    <profile>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+
+      <id>integration_tests</id>
+
+      <properties>
+        <bundleLocation>${basedir}/../../python/</bundleLocation>
+        <runName>end_to_end_test</runName>
+      </properties>
+
+      <build>
+        <plugins>
+
+          <plugin>
+            <artifactId>exec-maven-plugin</artifactId>
+            <groupId>org.codehaus.mojo</groupId>
+            <executions>
+              <execution><!-- Run our version calculation script -->
+                <id>generate Python models</id>
+                <phase>pre-integration-test</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>${basedir}/scripts/generate_model.sh</executable>
+                  <arguments>
+                    <argument>${runName}</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>2.22.0</version>
+            <executions>
+              <execution>
+                <phase>integration-test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <systemPropertyVariables>
+                    <bundleLocation>${bundleLocation}</bundleLocation>
+                    <runName>${runName}</runName>
+                  </systemPropertyVariables>
+
+                  <excludes>
+                    <exclude>TensorFlowInferenceTest</exclude>
```

Comment:
why is this "excluded"?

Reply:
good catch, this is a leftover.
But I think we would like to clean up those test in the near future.